test/index.js

```diff
@@ -156,6 +156,72 @@ describe('index', function() {
       expect(middleware).to.be.a('function');
       expect(middleware).to.have.length(3);
     });
+
+    function makeReq(props) {
+      return Object.assign({}, {url: 'http://example.com/path', path: '/path'}, props)
+    }
+
+    function makeRes(props) {
+      return Object.assign({}, {locals: {}, statusCode: 200, end: ()=>{}}, props)
+    }
+
+    it('should patch res.end and emit an info log instance for a 200 response', function(done) {
```

Comment:
These new tests could help raise our confidence level a bit if we decaffeinate winex.js.